CHANGELOG.md

```diff
@@ -7,10 +7,13 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 ## Master
 
 ##### Enhancements
+* Multi Pod Project Generation Support.  
```

Comment:
`Add support for splitting the pods product into a subproject per-pod, enabled by setting the xxx installation option to true`

Reply:
Added comment

Reply:
still needs a line between "Enchancements" title and first entry

Reply:
Ah my bad. I thought your previous comment was about a newline after `Multi Pod Project Generation` support which also needed a newline.